specification/compute/resource-manager/Microsoft.Compute/DiagnosticRP/preview/2025-02-01-preview/diagnostic.json

```diff
@@ -1053,13 +1107,39 @@
           ],
           "x-ms-enum": {
             "name": "FilterType",
-            "modelAsString": false
-          }
+            "modelAsString": false,
```

Comment:
ditto, suggestion for all modelasstring

Reply:
AS of now no SDK will be generated and as we plan on SDK it will be newer version. So at that point will make this change.